### PR TITLE
Compatibility patches for Star crafter's armory, vanilla XCOM weapons, and infinity rim: Ariadna

### PR DESCRIPTION
**Pull request by CMDR-Bill-Doors**
## Additions

- Patches for Star crafter's armory
- Patchs, ammo sets and ammo sprite for Infinity rim: Ariadna
- Patches and ammo sets for vanilla XCOM weapons

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] with and without patched mod loaded
- [ ] Playtested a colony

### CODE REVIEW

**Comment by databobek** [comment]
I've playtested a colony and didn't find any issues

**Comment by albertbokor** [comment]
CMC-300 breaks ui, it's unequipped item's infopanel is empty and breaks render of character editor ui and error log when equipped

**Comment by albertbokor** [comment]
irrelevant sidenote: Ariadna textures look very fried, who would want to play with this?

**Review by albertbokor — DISMISSED** [review]

File: Patches/Star Crafter's Armory/CMC300.xml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+						<PsychicSensitivity>-0.2</PsychicSensitivity>
+						<AimingAccuracy>0.15</AimingAccuracy>
+						<SmokeSensitivity>-1</SmokeSensitivity>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/equippedStatOffsets/ToxicSensitivity</xpath>
+					<value>
+						<ToxicSensitivity>-0.5</ToxicSensitivity>
```
Comment (N7Huntsman) [inline Patches/Star Crafter's Armory/CMC300.xml:148]: This tag is no longer used--it should be `<ToxicEnvironmentResistance>0.5</ToxicEnvironmentResistance>` instead.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
- The vodka bottle from Ariadna has a melee attack that needs patched.

**Review by SamaelGray — APPROVED** [review]

**Pull request merged**